package.json

```diff
@@ -58,7 +58,8 @@
     "@types/superagent": "^4.1.10"
   },
   "volta": {
-    "node": "18.16.0"
+    "node": "18.16.0",
+    "npm": "9.9.2"
```

Comment:
**nitpick**: this is leftover code from debugging and we should probably just bump Node instead of pinning a separate npm version but we can fix that later idm

Reply:
eh whatever